examples/puppet_headnode/manifests/static/boot_notify.py

```diff
@@ -1,10 +1,10 @@
-#!/usr/bin/env python
+import re
+import os
+
+# !/usr/bin/env python
```

Comment:
What tool is complaining about a shebang?  I just wrote a test script with a shebang, followed by an import, and ``flake8`` reports no pep8 problems